docs/user_manual.md

```diff
@@ -1098,18 +1136,18 @@ duplicate expressions on the LHS and RHS of the rewriting.
 ### Macros and Aliases
 
 A rule can be tagged with the `macro`, `alias`, `macro-rec`, or `alias-rec`
-attributes. In all cases, what this signifies is that this is a macro rule.
-Macro rules are applied statically during compilation on all terms that they
-match, and statically before program execution on the initial configuration.
-Currently, macros are required to not have side conditions, although they can
-contain sort checks.
-
-When a rule is tagged with the `alias` attribute, it is also applied statically
-in reverse prior to unparsing on the final configuration. Note that a macro can
-have unbound variables in the right hand side. When such a macro exists, it
-should be used only on the left hand side of rules, unless the user is
-performing symbolic execution and expects to introduce symbolic terms into the
-subject being rewritten.
+attributes. In all cases, what this signifies is that this is a macro production.
+Macro rules are rules where the top symbol of the left-hand-side are macro
+labels. Macro rules are applied statically during compilation on all terms that
+they match, and statically before program execution on the initial configuration.
+Currently, macro rules are required to not have side conditions, although they
+can contain sort checks.
+
+`alias` rules are also applied statically in reverse prior to unparsing on the
+final configuration. Note that a macro rule can have unbound variables in the
+right hand side. When such a macro exists, it should be used only on the left
+hand side of rules, unless the user is performing symbolic execution and expects
+to introduce symbolic terms into the subject being rewritten.
```

Comment:
`macro` should be added to productions now, not rules.
This text needs to be updated.

Reply:
That's the update I have made. `macro` is applied to productions, and a "macro rule" is a rule that has a macro as the toy symbol of the LHS. That's what it says: "macro rules are rules where the top symbol of the LHS is a macro production." in the sentence at the begcinning of this paragraph.

Reply:
> A rule can be tagged with the `macro`, `alias`, `macro-rec`, or `alias-rec`

Reply:
Ah thank you, I missed that one. Fixed.